mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
Something like this may be more concise.

```python
def mayafpsconverter(fps):
  return {
    None: "",
    "": "",
    "15": "game",
    "24": "film",
    "25": "pal",
    ...
  }.get(fps, fps + "fps")
```

No need to check whether the variable is a string, environment variables can only be string.